roles/mistral/defaults/main.yml

```diff
@@ -1,4 +1,4 @@
-mistral_version: 0.13
+mistral_version: 1.3.2-99
```

Comment:
`1.4.0` is latest stable version from https://packagecloud.io/StackStorm/stable. Let's stick to some revision from that version.

---

On related note, I would prefer to avoid such pinning in default config values. So if version is not explicitly set, playbook installs latest stable by default (with accordance with `st2` version).

Anyway, it's something to rethink later. For now it's good enough.